src/node.rs

```diff
@@ -1,3 +1,4 @@
+use crate::gossip::{read_peer_info, save_peer_connection_info};
```

Comment:
Nit: drop into the `crate` section below

Reply:
Yeah seems our IDEs handle imports differently, yours seems like it does it differently than most other projects. Will change the imports for now but we should try to configure our stuff to work the same (or at least get rustfmt to do it)

Reply:
Yeah my IDE doesn't do it in one big one either, I do that manually. Perhaps there is a `rustfmt` that does it properly. Or maybe I'm wrong.